py/torch_tensorrt/dynamo/_exporter.py

```diff
@@ -228,7 +228,10 @@ def create_trt_exp_program(
     and constructs an Exported Program object with the new IO node names and state_dict
     """
     input_nodes = [node for node in gm.graph.nodes if node.op == "placeholder"]
-    output_nodes = [node for node in gm.graph.nodes if node.op == "output"]
+    # output_nodes = [node for node in gm.graph.nodes if node.op == "output"]
+    # graph = [placeholder, conv1, relu, trt_node, getitem, output]
+    # output_nodes[0].args[0]
+    output_nodes = list(gm.graph.nodes)[-1].args[0]
```

Comment:
This will now contain a list of nodes which output Tensors, as opposed to the single output `Node`. Is this the intended behavior here?

Additionally, do we need to consider or include an assertion for graphs which have no outputs?

Reply:
Yes, that's the intended behavior. 
I added an assert output_nodes as a safe check, but I haven't found graphs without an output node implicitly added. Did you encounter them?

Reply:
No, I have not encountered such a case. I think every graph likely needs an output node, and any graph without an output node or with an empty `args` for its output node would be invalid, so I think the assert should suffice, thanks for the clarification.